vxyowsup/whatsapp.py

```diff
@@ -1,10 +1,26 @@
 # -*- test-case-name: vumi.transports.whatsapp.tests.test_whatsapp -*-
 from twisted.internet import defer
+from twisted.internet.threads import deferToThread
 
 from vumi.transports.base import Transport
 from vumi.config import ConfigText
 from vumi import log
 
+from yowsup.layers.network                             import YowNetworkLayer
+
+from yowsup.common import YowConstants
+from yowsup.layers import YowLayerEvent
+from yowsup.stacks import YowStack, YowStackBuilder
+from yowsup import env
+
+import sys, argparse, yowsup, logging
+
+from yowsup.layers.interface                           import YowInterfaceLayer, ProtocolEntityCallback
+from yowsup.layers.protocol_messages.protocolentities  import TextMessageProtocolEntity
+from yowsup.layers.protocol_receipts.protocolentities  import OutgoingReceiptProtocolEntity
+from yowsup.layers.protocol_acks.protocolentities      import OutgoingAckProtocolEntity
```

Comment:
We tend to only leave on space before `import`, although this does make things quite readable.